ci/jenkins/id_rsa

```diff
@@ -0,0 +1,38 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
Why do we need to checkin private key? is it safe in public repo

Reply:
will generate when it is running on the jenkins node